spec/models/state_id_spec.rb

```diff
@@ -45,5 +45,11 @@
         }.to change(state_id, :expiration_date).to(nil)
       end
     end
+    context "when a long dash is in the id number" do
+      it "removes the dash" do
+        state_id.update(id_number: "MD —123–456")
+        expect(state_id.id_number).to eq("MD 123456")
+      end
+    end
```

Comment:
is there a test example of a regular dash (not em-dash or en-dash) that _will_ be accepted?

Reply:
good call I did it